data/json/items/tool/med.json

```diff
@@ -322,7 +324,8 @@
   {
     "id": "oxygen_tank",
     "type": "TOOL",
-    "name": { "str": "oxygen tank" },
+    "name": { "str": "oxygen delivery kit" },
+    "category": "tools",
```

Comment:
I don't think it needs a category of ``tools`` specified if it's already ``type: TOOL``. Could you check that? I'm positive this is redundant.